devito/ops/types.py

```diff
@@ -104,3 +105,13 @@ def as_coeff_Add(self):
         return sympy.S.Zero, self
 
     __repr__ = __str__
+
+
+class OpsStencil(basic.LocalObject):
+
+    def __init__(self, name, *args, **kwargs):
+        super().__init__(name, np.void, *args, **kwargs)
+
+    @property
+    def _C_typename(self):
+        return "ops_stencil"
```

Comment:
This should be in the dictionary `namespace`.

Reply:
probably yes